### ISSUE TITLE
Partial builds

### ISSUE THREAD

**Issue by jasonmunro** [fabiospampinato/pacco#1]
Hello!
I'm interested in doing a partial/custom build of a project that uses pacco (https://github.com/kenwheeler/cash). Specifically I would like to skip including certain methods I don't use, which the docs suggest is possible. I have no idea how to configure the pacco.json file to do so. If you could provide some guidance it would be greatly appreciated.

Thanks for your time!

**Comment by fabiospampinato** [comment]
Yes this is something not well documented. You can find more info about this [here](https://github.com/kenwheeler/cash/blob/master/docs/partial_builds.md).

Basically you'll have to add a few lines to `pacco.json`. The way you can exclude for instance the `extend` method, which is not used internally but it's only provided for better plugins compatibility is by adding the following to `pacco.json`:

```
  "components": {
    "core/extend": false
  }
```

Basically under the `"component"` key you can list some relative paths that you'd like to exclude, so you could also write:

```
  "components": {
    "effects": false,
    "forms/serialize": false,
    "manipulation/empty.js": false
  }
```

And that's it. If there will be any dependencies problems (say you are not including `collection/each`, but you're including other files that need `collection/each`) pacco should throw an error.
